.github/workflows/ci.yaml

```diff
@@ -0,0 +1,60 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+
+      - name: Enable Corepack
+        run: corepack enable
+
+      - name: Install dependencies
+        run: yarn install --immutable
+
+      - name: Diff openapi.json with main
+        id: diff_openapi
+        run: |
+          git fetch origin main
+          if git diff --quiet origin/main -- ./openapi.json; then
+            echo "changed=true" >> $GITHUB_OUTPUT
+          fi
```

Comment:
I'm definitely not a scripting expert, but doesn't `if git diff --quiet origin/main -- ./openapi.json` output `0` if there are no difference and `1` if there are differences? ([docs](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---exit-code))

`0` is success so I think this conditional might be setting changed to `true` when there are no changes - which we want the opposite?